standardtracer/span.go

```diff
@@ -45,14 +56,14 @@ func (s *spanImpl) LogEventWithPayload(event string, payload interface{}) {
 }
 
 func (s *spanImpl) Log(ld opentracing.LogData) {
-	s.lock.Lock()
-	defer s.lock.Unlock()
+	s.Lock()
+	defer s.Unlock()
 
 	if ld.Timestamp.IsZero() {
 		ld.Timestamp = time.Now()
 	}
 
-	s.raw.Logs = append(s.raw.Logs, &ld)
+	s.raw.Logs = append(s.raw.Logs, ld)
```

Comment:
like what? The only thing that would ruin it are many large fields. If we're adding more pointers (strings, ...) then the actual struct should remain fairly small.